webrtc/asr_server_webrtc.py

```diff
@@ -59,12 +65,20 @@ async def stop(self):
     async def __run_audio_xfer(self):
         while True:
             frame = await self.__track.recv()
-            print ("Got frame", frame)
             frame = self.__resampler.resample(frame)
             data = frame.to_ndarray()
-            response = await loop.run_in_executor(pool, process_chunk, self.__recognizer, data.tobytes())
-            print ("Sending response", response)
-            self.__channel.send(response)
+
+            max_frames_len = 10000
+            global dataframes
```

Comment:
Thank you for your pull request, I'm happy Vosk started to work for you. Honestly this part is questionable since you use global variable you'll have troubles with multiple connections. You need to put dataframes into local variable. Also, 10000 is too much, something like 8000 make sense.

Reply:
Thanks for your feedback :) 
In this commit ,
https://github.com/milochen0418/vosk-server/commit/dea673254abdeaa73cc4a801d84d586cf2be0dd0 
I put dataframes into local variable and set max_frames_len for 8000.